.gitignore

```diff
@@ -64,10 +64,11 @@ DerivedData
 *.ipa
 
 # Swift Package Manager
-*/.build
```

Comment:
If this is deleted, need to add `ReleaseTooling/*.build`

Reply:
Have reverted and add .build for the tool directory `code_coverage_report/generate_code_coverage_report`